#64 move 'charge' transactions to 'authorization' transactions according to the adyen best practices. (#85)
commercetools/commercetools-adyen-integration: extension/docs/IntegrationGuide.md

Otherwise shopper might continue with further payment steps.

1. **Get available payment methods**  

>     > First part of the payment, the shopper needs to view the list of available payment methods before entering the payment details, please follow the steps below for getting available payment methods:

Reviewer: ```suggestion
    > First part of the payment, the shopper needs to view the list of available payment methods before entering the payment details. Please follow the steps below for getting available payment methods:
```

    1. [Create/update a CTP Payment](https://docs.commercetools.com/http-api-projects-payments) with following properties:
        - `Payment.paymentMethodInfo.method = empty or undefined`   
        - `Payment.custom.fields.countryCode != null` - set the country of a shopper. Please [consult with Adyen](https://docs.adyen.com/api-explorer/#/PaymentSetupAndVerificationService/v41/paymentMethods) for the right format.  
    1. Extension module will make a [request](https://docs.adyen.com/checkout/api-integration/#step-1-get-available-payment-methods) to Adyen API and the response will be saved in interface interaction with `type='ctp-adyen-integration-interaction'` and `fields.type='getAvailablePaymentMethods'` as stringified JSON.
    1. Before presenting the payment methods in the response from Adyen, please check Extension module documentation for supported payment methods.
1. **Continue with one of the supported payment methods**

>     > As a second part, after getting the list of available payment methods, shopper will select a payment method and make a payment, please follow the steps below for processing the payment with extension module:

Reviewer: ```suggestion
    > As a second part, after getting the list of available payment methods, the shopper will select a payment method and make a payment. Please follow the steps below for processing the payment with the extension module:
```

Otherwise shopper might continue with further payment steps.

1. **Get available payment methods**  

>     > First part of the payment, the shopper needs to view the list of available payment methods before entering the payment details. Please follow the steps below for getting available payment methods:

Reviewer: ```suggestion
    > As first part of the payment, the shopper to select a payment method out of the list of available payment methods before entering the payment details. Please follow the steps below for getting available payment methods:
```

    1. [Create/update a CTP Payment](https://docs.commercetools.com/http-api-projects-payments) with following properties:
        - `Payment.paymentMethodInfo.method = empty or undefined`   
        - `Payment.custom.fields.countryCode != null` - set the country of a shopper. Please [consult with Adyen](https://docs.adyen.com/api-explorer/#/PaymentSetupAndVerificationService/v41/paymentMethods) for the right format.  
    1. Extension module will make a [request](https://docs.adyen.com/checkout/api-integration/#step-1-get-available-payment-methods) to Adyen API and the response will be saved in interface interaction with `type='ctp-adyen-integration-interaction'` and `fields.type='getAvailablePaymentMethods'` as stringified JSON.
    1. Before presenting the payment methods in the response from Adyen, please check Extension module documentation for supported payment methods.
1. **Continue with one of the supported payment methods**

>     > As a second part, after getting the list of available payment methods, the shopper will select a payment method and make a payment. Please follow the steps below for processing the payment with the extension module:

Reviewer: ```suggestion
    > As a second part, after selecting a payment method and the user triggers a payment. Please follow the steps below for processing the payment with the extension module:
```
